ml_perf/flags/9/selfplay.flags

```diff
@@ -5,10 +5,12 @@
 
 --engine=tf
 --num_games=2048
---parallel_games=2048
 --num_readouts=160
---virtual_losses=8
---value_init_penalty=2
+--value_init_penalty=0.2
 --holdout_pct=0.03
 --disable_resign_pct=0.1
 --resign_threshold=-0.99
+
+# Device-specific selfplay flags.
+--parallel_games=2048
+--virtual_losses=8
```

Comment:
Is `virtual_losses` going to be fixed for MLPerf or are participants free to tune it? The comment suggests the latter, but I would have expected the former to be the case, since it effects the read depth.

Reply:
it effects read depth less than we might think with tree reuse.  I think it should be tunable; the actual effect on the pipeline seems very small -- v12 vs v10 we only changed the vlosses and it had a very mild effect on overall strength.  I think we may want to specify a maximum;